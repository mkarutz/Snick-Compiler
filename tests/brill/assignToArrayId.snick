proc main ()
    int a[0..9];

    a := 2;
end
