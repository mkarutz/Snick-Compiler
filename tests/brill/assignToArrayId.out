Failure: Expected subscript following array identifier: a
Compilation Failed
